controller/space_iterations_test.go

```diff
@@ -268,6 +284,24 @@ func (rest *TestSpaceIterationREST) TestWICountsWithIterationListBySpace() {
 	fmt.Println("iteration2 id = ", iteration2.ID)
 	assert.NotEqual(rest.T(), uuid.UUID{}, iteration2.ID)
 
+	childOfIteration2 := iteration.Iteration{
+		Name:    "Sprint 2.1",
+		SpaceID: spaceInstance.ID,
+		Path:    append(iteration2.Path, iteration2.ID),
+	}
+	iterationRepo.Create(rest.ctx, &childOfIteration2)
+	fmt.Println("iteration2 id = ", childOfIteration2.ID)
```

Comment:
I missed that and now I have removed print statements  https://github.com/almighty/almighty-core/pull/1151/commits/fb439257f1ee09bebdf4b186fb15cbda9824855a